core/src/com/unciv/ui/popups/options/MultiplayerTab.kt

```diff
@@ -101,7 +102,7 @@ private fun addMultiplayerServerOptions(
 ) {
     val settings = optionsPopup.settings
 
-    val connectionToServerButton = "Check connection to server".toTextButton()
+    val connectionToServerButton = "Check connection".toTextButton()
```

Comment:
This requires adding this text in template.properties, so it'll be available for translators 👍🏿

Reply:
[Is this all right?](https://github.com/yairm210/Unciv/pull/12650/commits/b476fabc59021ca7dbe5969245d5badce3501734) Guessing the original string doesn't need be kept since it's not used elsewhere in the game itself.